### PR TITLE
Update Docs for the JDK 11 Migration

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Update docs for the JDK 11 migration.
> Fixes #

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: swan-lake/learn/installing-ballerina.md [hunk 1]
```diff
@@ -92,8 +92,8 @@ Alternatively, follow the instructions below to build Ballerina from the source.
 ### Setting Up the Prerequisites
 
 You need to download and install the below to build the Ballerina modules.
-1. Java SE Development Kit (JDK) version 8 (from one of the following locations) 
-    - [Oracle](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+1. Java SE Development Kit (JDK) version 11 (from one of the following locations) 
+    - [Oracle](https://www.oracle.com/java/technologies/javase-downloads.html)
```
Comment (keizer619) [inline swan-lake/learn/installing-ballerina.md:96]: Shall we change the order of OpenJDK?

**Comment by praneesha** [comment]
Updates of the "ballerina-lang" repo are done via [1].

[1] https://github.com/ballerina-platform/ballerina-lang/pull/26334

File: swan-lake/learn/installing-ballerina.md (hunk 1 above)
Comment (praneesha) [reply swan-lake/learn/installing-ballerina.md]: Done.

File: swan-lake/learn/installing-ballerina.md [hunk 2]
```diff
@@ -92,10 +92,10 @@ Alternatively, follow the instructions below to build Ballerina from the source.
 ### Setting Up the Prerequisites
 
 You need to download and install the below to build the Ballerina modules.
-1. Java SE Development Kit (JDK) version 8 (from one of the following locations) 
-    - [Oracle](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
-    - [OpenJDK](http://openjdk.java.net/install/index.html)
-    >**Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed JDK.
+1. Open JDK 11 ([Adopt OpenJDK](https://adoptopenjdk.net/) or any other Open JDK distribution)
```
Comment (anupama-pathirage) [inline swan-lake/learn/installing-ballerina.md:95]:
```suggestion
1. OpenJDK 11 ([Adopt OpenJDK](https://adoptopenjdk.net/) or any other OpenJDK distribution) 
```

File: swan-lake/learn/installing-ballerina.md [hunk 3]
```diff
@@ -92,10 +92,10 @@ Alternatively, follow the instructions below to build Ballerina from the source.
 ### Setting Up the Prerequisites
 
 You need to download and install the below to build the Ballerina modules.
-1. Java SE Development Kit (JDK) version 8 (from one of the following locations) 
-    - [Oracle](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
-    - [OpenJDK](http://openjdk.java.net/install/index.html)
-    >**Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed JDK.
+1. Open JDK 11 ([Adopt OpenJDK](https://adoptopenjdk.net/) or any other Open JDK distribution) 
+    
+>**Info:** You can also use [Oracle JDK](https://www.oracle.com/java/technologies/javase-downloads.html). Set the JAVA_HOME environment variable to the path name of the directory into which you installed JDK.
```
Comment (anupama-pathirage) [inline swan-lake/learn/installing-ballerina.md:97]: This should be indented.

Comment (praneesha) [reply swan-lake/learn/installing-ballerina.md]: Fixed.

**Review by keizer619 — APPROVED** [review]

**Pull request merged**